support-frontend/test/codecs/ProductTypeDecoderTest.scala

```diff
@@ -0,0 +1,104 @@
+package codecs
+
+import com.gu.i18n.Currency
+import com.gu.support.catalog.{Collection, Sunday}
+import com.gu.support.workers.ProductType._
+import com.gu.support.workers._
+import io.circe.parser.parse
+import org.scalatest.{MustMatchers, WordSpec}
+
+class ProductTypeDecoderTest extends WordSpec with MustMatchers {
```

Comment:
I added this after I needed to swap the order in `ProductType`'s decoder because my changes to support print subs were initially creating digital packs 🙈

Reply:
It would be good to use the `SerialisationTestHelpers` trait for these tests. See an example in: https://github.com/guardian/support-frontend/blob/79f05140561d2e1fb571b90871654711a3aec96e/support-models/src/test/scala/com/gu/support/promotions/SerialisationSpec.scala#L12

Reply:
💯 for adding a test for this though!

Reply:
Thanks for pointing me to that. Definitely nicer. https://github.com/guardian/support-frontend/pull/1561/commits/5f2d09d550087fdb9d3c54afed5310b8b8b6b8a3